include/Dialect/Secret/Transforms/Passes.td

```diff
@@ -60,4 +60,14 @@ def SecretMergeAdjacentGenerics : Pass<"secret-merge-adjacent-generics"> {
   let dependentDialects = ["mlir::heir::secret::SecretDialect"];
 }
 
+def SecretConstantPassThroughGeneric : Pass<"secret-constant-pass-through-generic"> {
```

Comment:
Two minor suggestions you can leave for a future ticket if desired:

- Make this pass apply to any op that has the <ConstantLike> trait
- rename the pass "secret-generic-absorb-constants"

Reply:
done and done! thank you! i like that name a whole lot better